File: 프로그래머스/unrated/181855. 문자열 묶기/문자열 묶기.py

def solution(strArr):
    group = {}
    
    for string in strArr:
        length = len(string)
        
        if length not in group:
            group[length] = 1
        else:
            group[length] += 1
    
    print(group)
    
    return max(group.values())

File: 프로그래머스/0/181855. 문자열 묶기/문자열 묶기.java
import java.util.Collections;
class Solution {
    public int solution(String[] strArr) {
        int answer = 0;
        int[] counts = new int[31];
        for(int i = 0; i < strArr.length; i++) {
            int len = strArr[i].length();
            counts[len] ++;
        }
        System.out.println(counts);
        
        return getMax(counts);
    }
    
    // 최대값 구하기
    public int getMax(int[] arr) {
        int max = 0;
        for(int i = 0; i < arr.length; i++) {
            if(arr[i] >= max)
                max = arr[i];
        }
        return max;
    }
}